index.bs

```diff
@@ -361,7 +361,70 @@ Report long tasks {#report-long-tasks}
 Security & privacy considerations {#priv-sec}
 ===============================================
 
-Long Tasks API adheres to the same-origin policy by including origin-safe attribution information about the source of the long task.
-There is a 50ms threshold for long tasks. Together this provides adequate protection against security attacks against browser.
+Long Tasks API adheres to the same-origin policy by including origin-safe attribution information about
+the source of the long task. There is a 50ms threshold for long tasks. Together this provides adequate
+protection against security attacks against the browser itself.
+
+The Long Tasks API provides timing information about the duration and type of tasks executed by the user,
+as well as attribution such as source code location for function calls. This could enable an attacker
+to perform side-channel timing attacks to guess user’s action, or identify the user. For example, a pattern of
+long script followed by a long render could be put together to guess user’s interaction with a social
+widget. Detailed function call attribution would be used to determine the user’s action.
+
+However, privacy related attacks are possible. While the API doesn’t introduce any new privacy attacks,
+it could make existing privacy attacks faster. Mitigations for this are possible and can be implemented
+as needed:
+
+* Further clamp the long task duration provided by the API to make attacks harder to exploit.
+* Limit the number of origins for which longtasks are exposed by the API, and obfuscate the attribution of
+    any tasks afterwards.
+* Allow dropping the culprit/attribution information after a certain threshold.
+* Add a built-in delay to the timing information exposed to make attacks dependent on longtask volume harder
+    to execute.
+
+What is Exposed to Observers? {#what-is-exposed}
+--------------------------------------------------------
+
+All observers within the top level page (i.e. all iframes in the page and the main frame) will receive
+notifications about presence of long tasks. We expose the start time of the task, its duration, and a
+pointer to the culprit frame. This information can already be observed today, and with higher resolution,
+using setTimeout. Observers in other different pages (tabs or windows) should not receive notifications,
```

Comment:
Does that include same-origin observers? e.g. if one example.com tab is opening another without noopener (and hence both are running on the same event loop), will one of them no know when the other is triggering LTs that freeze its user input as well?

Reply:
Oh good question... My understanding is that we do not report the longtask to the other tabs, though arguably we may want to change this behavior if we think this happens often, as that can be useful information.

Reply:
Opened https://github.com/w3c/longtasks/issues/88 to keep track of this